libs/consumer/lib/job_listings.dart

```diff
@@ -3,8 +3,7 @@ import 'package:flutter_redux/flutter_redux.dart';
 import './job_details.dart';
 import 'package:redux/redux.dart';
 import 'package:redux_comp/redux_comp.dart';
-import 'package:flutter/rendering.dart';
-
+//import 'package:flutter/rendering.dart';
```

Comment:
nit: Should we not be removing these rather than commenting?

Reply:
Yes you are right there. I did that because it was not my code and I think the front end team plans on using them. Just commented it out so they know what I changed it. It was failing the pipeline, but next time I will make sure to ask them in case the commented out stuff accumulates